infra/base-images/base-runner/bad_build_check

```diff
@@ -371,6 +371,22 @@ function check_architecture {
 
 function main {
   local FUZZER=$1
+
+  # Move the directory the fuzzer is located in to somewhere that doesn't exist
+  # on the builder to make it more likely that hardcoding /out fails here (since
+  # it will fail on ClusterFuzz).
+  FUZZER=$(realpath $FUZZER)
+  local INITIAL_FUZZER_DIR=$(dirname $FUZZER)
+
+  local FUZZER_DIR=/tmp/mnt/scratch0/clusterfuzz/bot/builds/build/revisions
+  rm -rf $FUZZER_DIR
+  # Make FUZZER_DIR's parents.
+  mkdir -p $FUZZER_DIR
+  # Move contents of INITIAL_FUZZER_DIR rather than the directory itself since
+  # usually (when it is /out) it is mounted and cannot be moved.
+  mv $INITIAL_FUZZER_DIR/* $FUZZER_DIR
```

Comment:
test_all can call multiple instances of bad-build-check, so this will fail after first iteration, cp -R ?

Reply:
I assumed the container gets torn down after bad_build_check is run.

cp is not enought because you can still load /out/my-dynamic-library.so

Reply:
I'm afraid we'll have to move it back and forth then, otherwise files will still be present in `/out`